docs/guides/writing_stubs.rst

```diff
@@ -93,6 +93,9 @@ Liskov substitutability or detecting problematic overloads.
 It may be instructive to examine `typeshed <https://github.com/python/typeshed/>`__'s
 `setup for testing stubs <https://github.com/python/typeshed/blob/main/tests/README.md>`__.
 
+To suppress type errors in stubs, use `# type: ignore` comments. Refer to the style guide for
```

Comment:
Could we make this a link? What style guide is it referring to?

Reply:
There's a "style guide" section on the page. I added a link to the appropriate sub-section.